tools/build/scanCode.py

```diff
@@ -25,85 +36,134 @@
-#   - files end with EOL
-#   - license headers everywhere applicable
+VERBOSE = False
+
+# Translatable messages (error and general)
+ERR_LICENSE = "file does not include required license header."
+ERR_SYMBOLIC_LINK = "file is a symbolic link."
+ERR_TABS = "line contains tabs."
+ERR_TRAILING_WHITESPACE = "line has trailing whitespaces."
+ERR_NO_EOL_AT_EOF = "file does not end with EOL."
+ERR_PATH_IS_NOT_DIRECTORY = "%s: %s is not a directory.\n"
+ERR_GENERAL = "There was an error."
+MSG_CHECKING_FILE = "Checking file[%s]..."
+MSG_CHECKS_PASSED = "All checks passed."
+MSG_SCRIPT_USAGE = "Usage: %s root_directory\n"
+MSG_ERROR_SUMMARY = "Summary: Scan detected %d error(s) in %d file(s)."
+
+
+def vprint(s):
+    """Conditional print (stdout)."""
+    if VERBOSE:
+        print s
+
 
-####
-# Exceptions: these paths are not subjected to the tests for one reason or another.
-####
 def exceptional_paths():
+    """List of paths not subjected to the scan tests."""
     return [
         "bin/wskadmin",
         "bin/wskdev",
         "tests/build/reports"
     ]
 
-####
-# Line checks. Check a file line-by-line and produce at most one error per line.
-####
+
 def no_tabs(line):
+    """Assert line does not contains a TAB characeter."""
     if re.match("\t", line):
-        return "line contains tabs"
+        return ERR_TABS
     else:
         return None
 
+
 def no_trailing_spaces(line):
+    """Assert line does not have trailing whitespace."""
     if len(line) > 0 and line[-1] == '\n':
         line = line[:-1]
 
     if re.match("""^.*\s$""", line):
-        return "line has trailing whitespaces"
+        return ERR_TRAILING_WHITESPACE
     else:
         return None
 
+
 def eol_at_eof(line):
+    """Assert line at End of File is an End of Line character."""
     if len(line) == 0 or line[-1] != '\n':
-        return "file does not end with EOL"
+        return ERR_NO_EOL_AT_EOF
     else:
         return None
 
-####
-# File checks. Check a file in its entirety and produce an arbitrary number of errors.
-####
-def has_block_copyright(path):
-    header = textwrap.dedent("""\
-        /*
-         * Copyright 2015-2016 IBM Corporation
-         *
-         * Licensed under the Apache License, Version 2.0 (the "License");
-         * you may not use this file except in compliance with the License.
-         * You may obtain a copy of the License at
-         *
-         * http://www.apache.org/licenses/LICENSE-2.0
-         *
-         * Unless required by applicable law or agreed to in writing, software
-         * distributed under the License is distributed on an "AS IS" BASIS,
-         * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-         * See the License for the specific language governing permissions and
-         * limitations under the License.
-         */
-        """)
+"""Declare approved softare license headers as strings."""
+
+LICENSE_APACHE_SOFTWARE_FOUNDATION = """\
```

Comment:
My understanding was that once the scancode tool is in, we can go ahead and move forward and replace all copyright headers with the Apache one.
Since the CREDITS file now contains the attribution to IBM.

Once we know there is no more IBM Copyright across all repos, then we can modify scancode again to only check for Apache header. Apache header doesn't have year